docs/unlocking_the_bootloader.md

```diff
@@ -1,17 +1,14 @@
-#### On unlocking the bootloader
+# On unlocking the bootloader
 
-First of all, this tool will not help you bypass any vender locked bootloaders in non-official ways.
+[!IMPORTANT] First of all, this tool will not help you bypass any vender locked bootloaders in non-official ways.
```

Comment:
To get the correct formatting, it should be like this:

```
> [!IMPORTANT]
> First of all, this tool will not help you bypass any vender locked bootloaders in non-official ways.
```

Reply:
I'll fix it